src/librustdoc/html/static/rustdoc.css

```diff
@@ -985,6 +992,45 @@ a.test-arrow:hover{
 	font-weight: 300;
 }
 
+.collapse-toggle {
+	font-weight: 300;
+	position: absolute;
+	left: -23px;
+	top: 0;
+}
+
+div[data-level="1"] > .collapse-toggle, div[data-level="2"] > .collapse-toggle {
+	font-size: 0.8em;
+	top: 5px;
+}
+
+.toggle-wrapper > .collapse-toggle {
+	left: -24px;
+	margin-top: 0px;
+}
+
+.toggle-wrapper {
+	position: relative;
+	margin-top: 0;
+}
+
+.toggle-wrapper.collapsed {
+	height: 25px;
+	transition: height .2s;
+	margin-bottom: .6em;
+}
+
+.collapse-toggle > .inner {
+	display: inline-block;
+	width: 1.2ch;
+	text-align: center;
+}
+
+.collapse-toggle.hidden-default {
+	position: relative;
+	margin-left: 20px;
+}
+
```

Comment:
It looks like these were accidentally reintroduced, perhaps in a rebase?

Reply:
Oups indeed!